src/channels/components/ChannelForm/ChannelForm.tsx

```diff
@@ -216,73 +176,79 @@ export const ChannelForm: React.FC<ChannelFormProps> = ({
             )}
             disabled={disabled}
-            label={intl.formatMessage({
-              id: "tV+Dcm",
-              defaultMessage: "Default country",
-            })}
+            label={intl.formatMessage(messages.defaultCountry)}
             choices={countries}
             name="defaultCountry"
             displayValue={selectedCountryDisplayName}
             value={data.defaultCountry}
             onChange={onDefaultCountryChange}
           />
-          <FormSpacer />
-          <Box display="flex" gap={1.5} alignItems="center">
-            <ControlledSwitch
-              data-test-id="order-settings-mark-as-paid"
-              disabled={disabled}
-              checked={
-                data.markAsPaidStrategy ===
-                MarkAsPaidStrategyEnum.TRANSACTION_FLOW
-              }
-              onChange={onMarkAsPaidStrategyChange}
-              name="markAsPaidStrategy"
-              label={
-                <span>
-                  <FormattedMessage
-                    defaultMessage='"Mark as paid" feature creates a'
-                    id="MDOw8D"
-                  />{" "}
-                  <Link
-                    href="https://docs.saleor.io/docs/3.x/developer/payments#processing-a-payment-with-payment-app"
-                    target="_blank"
-                    rel="noopener noreferer"
-                  >
-                    <FormattedMessage
-                      defaultMessage="Transaction"
-                      id="1+ROfp"
-                    />
-                  </Link>{" "}
-                  <FormattedMessage
-                    defaultMessage="- used by Payment Apps"
-                    id="Fqe4aB"
-                  />
-                </span>
-              }
-              secondLabel={
-                <span>
-                  <FormattedMessage
-                    defaultMessage="If left unchecked it creates a"
-                    id="hHv0ih"
-                  />{" "}
-                  <Link
-                    href="https://docs.saleor.io/docs/3.x/developer/payments#payment-plugin"
-                    target="_blank"
-                    rel="noopener noreferer"
-                  >
-                    <FormattedMessage defaultMessage="Payment" id="NmK6zy" />
-                  </Link>{" "}
-                  <FormattedMessage
-                    defaultMessage="- used by Payment Plugins"
-                    id="50lR2F"
-                  />
-                </span>
-              }
-            />
-            <PreviewPill />
-          </Box>
-        </CardContent>
-      </Card>
+        </Box>
+        <Box paddingX={6}>
+          <Input
+            name="deleteExpiredOrdersAfter"
+            value={data.deleteExpiredOrdersAfter}
+            error={!!formErrors.deleteExpiredOrdersAfter}
+            type="number"
+            label="TTL"
+            onChange={onChange}
+            min={0}
+            max={120}
+            __height={12.5}
```

Comment:
**suggestion**: can you add todo comment

So we won't forget about removing those when we migrate this page fully to new MacawUI